db/queries.py

```diff
@@ -3,44 +3,68 @@
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    return list(Message.objects.filter(text__icontains=word))
 
 
 def get_untitled_chats() -> list[Chat]:
-    pass
+    return Chat.objects.filter(title__startswith="Untitled")
 
 
 def get_users_who_sent_messages_in_2015() -> list[str]:
-    pass
+    return list(Message.objects.filter(
+        sent__year=2015).values_list("user__first_name",
+                                     "user__last_name"))
 
 
 def get_actual_chats() -> list[Chat]:
-    pass
+    return list(Chat.objects.filter(message__sent__year__gt=2020))
 
 
 def get_messages_contain_authors_first_name():
-    pass
+    return list(Message.objects.filter(
+        text__icontains=F("user__first_name")))
```

Comment:
if i am not mistaken, only one function  `get_users_who_sent_messages_starts_with_m_or_a`  is case-insensitive